web/src/Setting.js

```diff
@@ -285,18 +285,25 @@ export function getAvatarColor(s) {
   return colorList[random % 4];
 }
 
+export const I18n = (text) => {
+  return (
+    <Translation>
```

Comment:
Why not continue to use `i18n`, `i` is lowercase.

Reply:
@sh1luo I agree

@WindSpiritSR plz change